soccer/src/soccer/strategy/agent/agent_action_client.cpp

```diff
@@ -170,7 +173,7 @@ void AgentActionClient::update_position(const rj_msgs::msg::PositionAssignment::
             next_position_ = std::make_unique<Goalie>(robot_id_);
             break;
         case 1:
-            next_position_ = std::make_unique<Defense>(robot_id_);
+            next_position_ = std::make_unique<Runner>(robot_id_);
```

Comment:
this is our bad for having incomplete documentation, but this switch-case is supposed to correlate exactly with the `enum` defined in `coach_node.hpp`. So runner would not be 1, but rather 3. The other statements are fine, they are robot ID